pages/data-privacy.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import Hero  from "../components/dataPrivacy/HeroSection";
+import styled from "styled-components";
+import { breakpoint } from "../frontend-config";
+
+const Headline = styled.h3`
+  font-size: 25px;
+  line-height: 29px;
+  margin-bottom: 12px;
+  margin-top: 30px;
+`;
+
+const Section = styled.div`
+  padding: 35px 75px 38px;
+  line-height: 1.5em;
+
+  @media (min-width: ${breakpoint("lg")}) {
+    padding: 40px 75px 62px;
+  }
+`;
+
+export default function Home() {
```

Comment:
Just realized a few of our pages are exported as `Home`. 
Not that it breaks anything, but I would vote to rename this function to match the file name (i.e. - "export default function DataPrivacy()". 
At least that is the convention I am used to...